core/server/services/settings/validate.js

```diff
@@ -51,6 +51,15 @@ _private.validateData = function validateData(object) {
         longForm.query[options.resourceKey || resourceKey] = {};
         longForm.query[options.resourceKey || resourceKey] = _.omit(_.cloneDeep(RESOURCE_CONFIG.QUERY[resourceKey]), 'alias');
 
+        // @NOTE: `data: author.foo` is not allowed currenlty, because this will make {{author}} available in the theme, which is deprecated (single author usage)
+        if (!RESOURCE_CONFIG.QUERY[resourceKey] ||
```

Comment:
This can probably switched with the upper line.

Reply:
will update

Reply:
If you're in there then maybe fix the comment typo "currenlty" whilst there but else no need :)

Reply:
👍